build/config.guess

```diff
@@ -634,7 +634,8 @@ EOF
 		sed 's/^		//' << EOF > "$dummy.c"
 		#include <sys/systemcfg.h>
 
-		main()
+		int
+		main ()
```

Comment:
I guess no need to "fix" it upstream, seems only for more exotic oses.

Reply:
If you mean the missing "void", it would be good to fix probably, but sending patches to these Savannah lists is very cumbersome. And this won't cause issues I think, yes.